arrow-libs/core/arrow-core-data/src/main/kotlin/arrow/core/Const.kt

```diff
@@ -1,41 +1,9 @@
 package arrow.core
 
-import arrow.Kind
-import arrow.KindDeprecation
 import arrow.typeclasses.Monoid
 import arrow.typeclasses.Semigroup
 
-@Deprecated(
-  message = KindDeprecation,
-  level = DeprecationLevel.WARNING
-) class ForConst private constructor() {
-  companion object
-}
-
-@Deprecated(
-  message = KindDeprecation,
-  level = DeprecationLevel.WARNING
-) typealias ConstOf<A, T> = arrow.Kind2<ForConst, A, T>
-
-@Deprecated(
-  message = KindDeprecation,
-  level = DeprecationLevel.WARNING
-) typealias ConstPartialOf<A> = arrow.Kind<ForConst, A>
-
-@Suppress("UNCHECKED_CAST", "NOTHING_TO_INLINE")
-@Deprecated(
-  message = KindDeprecation,
-  level = DeprecationLevel.WARNING
-)inline
-fun <A, T> ConstOf<A, T>.fix(): Const<A, T> =
-  this as Const<A, T>
-
-@Deprecated(
-  message = KindDeprecation,
-  level = DeprecationLevel.WARNING
-) fun <A, T> ConstOf<A, T>.value(): A = this.fix().value()
-
-data class Const<A, out T>(private val value: A) : ConstOf<A, T> {
```

Comment:
I think we shouldn't remove this part, should we? Just the `ConstOf<A, T>` extension

Reply:
Only the `ConstOf<A, T>` extensions has been removed? 🤔 Am I missunderstanding your comment?

Reply:
Nevermind, I didn't see you were adding 
```
data class Const<A, out T>(private val value: A) {
```